core/src/main/java/de/leonkoth/blockparty/listener/PlayerLeaveArenaListener.java

```diff
@@ -49,7 +49,7 @@ public void onPlayerLeaveArena(PlayerLeaveArenaEvent event) {
 
         if(arena.getArenaState() == ArenaState.LOBBY && !arena.getPhaseHandler().getLobbyPhase().isRunning())
         {
-            this.blockParty.getDisplayScoreboard().setScoreboard(0,0,arena);
+            player.setScoreboard(Bukkit.getScoreboardManager().getNewScoreboard());
```

Comment:
`getMainScoreboard()` is better, no need to create a new scoreboard every time.

Reply:
Committed changes to c95f024
It worked fine in the test environment.